opm/simulators/flow/ReservoirCouplingMaster.cpp

```diff
@@ -101,6 +132,7 @@ void ReservoirCouplingMaster::spawnSlaveProcesses(int argc, char **argv) {
         );
         this->master_slave_comm_.push_back(std::move(master_slave_comm));
         this->slave_names_.push_back(slave_name);
+        this->num_slaves_++;
```

Comment:
DO we need this member? Might be implicitly given by `this->slave_names_.size()`